docs/operators/jdbcPollInputOperator.md

```diff
@@ -0,0 +1,155 @@
+JDBC Poller Input Operator
+=============
+
+## Operator Objective
+This operator scans jdbc table in parallel fashion. The jdbc table is table in jdbc complaint database. To provide parallelism during read each instance of Apex input operator also known as operator partition reads one portion of jdbc table. Based on requested operator instances number of non-polling static partitions are created to scan table portitions. The non-polling partition will scan the table portion only once. Additional single operator partition is created to scan newly added data incrementally. 
```

Comment:
An important part of this operator is the use of a worker thread to asynchronously fetch data from the DB and pass it to the operator thread for further processing. So we need to mention that right up front and also explain why this is needed in the first place. This paragraph overall is very difficult to understand. Suggest rewriting to provide a broader context for the problems addressed by this operator, something like this:

    Input operators often face a couple of problems:
    1. As discussed in [Development Best Practices](https://github.com/apache/apex-core/blob/master/docs/development_best_practices.md),
        the operator callbacks such as `beginWindow()`, `beginWindow()`, `emitTuples()`, etc.
        which are invoked by the main operator thread
        are required to return quickly, well within the default streaming window duration of
        500ms. This requirement can be an issue when retrieving data from slow external systems
        such as databases or object stores: if the call takes too long, the platform will deem
        the operator blocked and restart it. Restarting will often run into the same issue
        causing an unbroken sequence of restarts.

    2. When a large volume of data is available from a single store that allows reading from
       arbitrary locations (such as a file or a database table), reading the data sequentially
       can be throughput limiting: Having multiple readers read from non-overlapping sections
       of the store allows any downstream parallelism in the DAG to be exploited better to
       enhance throughput. For files, this approach is used by the file splitter and block
       reader operators in the Malhar library.

    The current operator addresses the first with an asynchronous worker thread which
    retrieves the data and adds it to an in-memory queue; the main operator thread
    dequeue tuples very quickly if data is available or simply return if not. The second is
    addressed in a way that parallels the approach to files by having multiple partitions
    read records from non-overlapping areas of the table. Additional details of how this is
    done are described below.

The text about "non-polling" partitions seems like implementation detail that can be moved to a later section.